lms/templates/login.html

```diff
@@ -4,11 +4,12 @@
 <%!
 from django.urls import reverse
 from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
 import third_party_auth
 from third_party_auth import provider, pipeline
 %>
 
-<%block name="pagetitle">${_("Log into your {platform_name} Account").format(platform_name=platform_name)}</%block>
+<%block name="pagetitle">${Text(_("Log into your {platform_name} Account")).format(platform_name=platform_name)}</%block>
```

Comment:
Note: This is equivalent to:
```
<%block name="pagetitle">${_("Log into your {platform_name} Account").format(platform_name=platform_name) | h}</%block>
```
And then, when we got page level `h` filter, it could just go back to what it was orginally.  You shouldn't need the HTML/Text calls unless you have HTML somewhere in there.  You can just use the `h` filter.  Also, the linter will ask you to clean-up/remove the `h` filters once the page-level filter is added, but it won't notice the extra `Text` calls, so it just makes it a little messier than necessary.

Reply:
Done